docs/index.md

```diff
@@ -31,7 +31,7 @@ Spine animations used to be part of the main Defold engine. Starting with Defold
     - Set this as the output suffix in the Spine Editor
 
 * Update the spine source files to latest version
-    - The new runtime is based on Spine 4.0+
+    - The new runtime is based on Spine 4.0.+
```

Comment:
I think we should write it exactly like it is done in the spine-c runtime:

https://github.com/EsotericSoftware/spine-runtimes/tree/4.0/spine-c#spine-version

It is too bad that there is no version mentioned in the source code. We could refer to that instead.

Reply:
Good point, changed it to "4.0.xx"